pkg/e2e/state/prometheus.go

```diff
@@ -26,6 +31,27 @@ var _ = ginkgo.Describe(clusterStateInformingName, func() {
 		// common for a running prometheus instance
 		cmd := promCollectCmd + " > " + runner.DefaultRunner.OutputDir + "/prometheus.tar.gz; err=$? ; if (( $err != 0 )) ; then exit $err ; fi"
 
+		poderr := wait.PollImmediate(2*time.Second, 5*time.Minute, func() (bool, error) {
+			podCount := 0
+			var podErr error
```

Comment:
I don't think you need this variable, though perhaps I'm missing something.

Reply:
Yup I guess we wouldn't be needing this variable.